hivemind/utils/grpc.py

```diff
@@ -207,6 +254,18 @@ def serialize_torch_tensor(tensor: torch.Tensor, compression_type=CompressionTyp
             size=array.shape,
             dtype=array.dtype.name,
             requires_grad=tensor.requires_grad)
+    elif compression_type == CompressionType.QUANTILE_8BIT:
```

Comment:
The compression type is named as 8BIT, but in fact we can easily change the number of bits. Shouldn't we make the name more general?

Reply:
As a counterargument: this specific compression is 8-bit and our protocol does not support parameterized compressions yet.